pkg/api/vm/formatter.go

```diff
@@ -228,28 +225,6 @@ func (vf *vmformatter) canStop(vm *kubevirtv1.VirtualMachine, vmi *kubevirtv1.Vi
 	return true
 }
 
-func canMigrate(vmi *kubevirtv1.VirtualMachineInstance) bool {
-	if vmi == nil {
-		return false
-	}
-
-	if !vmi.IsRunning() {
```

Comment:
this condition is very important, only when vmi is running, it has the possibility to migrate

Reply:
check https://github.com/harvester/harvester/issues/7140#issuecomment-2607710331, both pending starting VM and stopped VM have `migrate` menu.